.github/workflows/skywalking.yaml

```diff
@@ -490,7 +490,7 @@ jobs:
               base: test/e2e-v2/cases/profiling/ebpf/offcpu/
               file: Dockerfile.file
               name: test/offcpu:test
-            runs-on: ubuntu-20.04
+            runs-on: ubuntu-24.04
```

Comment:
Due to https://github.com/actions/runner-images/issues/11101 ubuntu 20.04 is not available since yesterday.